src/components/chip/chip.tsx

```diff
@@ -315,5 +327,31 @@ export class Chip implements ChipInterface {
         }
 
         return <limel-linear-progress indeterminate={true} />;
-    };
+    }
+
+    private renderProgressBar() {
+        if (!this.progress) {
+            return;
+        }
+
+        const percentage = 100;
+        const currentPercentage =
+            (this.progress * percentage) / percentage + '%';
+        const progress = Math.round(this.progress * 10) / 10; // eslint-disable-line no-magic-numbers
+
+        return (
+            <div
+                role="progressbar"
+                aria-label="%"
+                aria-valuemin="0"
+                aria-valuemax="100"
+                aria-valuenow={this.progress}
+                style={{
+                    '--limel-chip-progress-percentage': currentPercentage,
+                }}
+            >
+                <span>{abbreviate(progress)}%</span>
```

Comment:
What's `abbreviate` used for here?

Reply:
I copied this from `limel-circular-progress`. I think sometimes we could get numbers like `12.23679` and wanted to convert them to 1 decimal this way. So the UI would display `12.2`

Reply:
Ah, yeah, that might be it. Let me give it a quick try 👍

Reply:
Even with `abbreviate` it can show one decimal, and without it, we still ensure it can never show more than one decimal by rounding the value on line 340:

<img width="506" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/d068a756-8a4e-43f0-817b-47887e5b0f46">

If we want to avoid ever showing decimals here, we can just adjust line 340 like this:

```diff
-        const progress = Math.round(this.progress * 10) / 10; // eslint-disable-line no-magic-numbers
+        const progress = Math.round(this.progress);
```

Wait… now that I look at that line… 🤨 😄

Reply:
Should we just go with `const progress = Math.round(this.progress);` on line 340 then?

Reply:
this works absolutely in this case, because we won't need to show any decimals at all actually.
So getting this UI 👇 which the previous code generates is actually useless in this context.
<img width="250" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/67e59d62-8af2-4d54-a528-982fe2123a37">
I'll add a fixup!

Reply:
What do you think about this? 
<img width="430" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/fde92876-e88e-4d17-bafa-d618df41ba31">

It also looks unnecessary. We expect the consumer to give us a number between 0 to 100 anyways, right?

Reply:
Added a fixup and simplified all of these.